src/styles/reset.scss

```
@@ -0,0 +1,21 @@
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+/* #region reset */
+/* body,
+div,
+span,
+h2,
+h3,
+p,
+a,
+img,
+strong {
+  margin: 0;
+  padding: 0;
+} */
+/* #endregion */
```

```suggestion
body,
h2,
h3,
p,
ul {
  margin: 0;
  padding: 0;
} 
```
looks like it should be uncomment, but some tags redundant, because don't have default margins or paddings.